TrackerViewExtension.cs

```diff
@@ -69,6 +69,8 @@ public void TrackerMenuItems()
             var sayHelloMenuItem = new MenuItem {Header = " ❓ About"};
             sayHelloMenuItem.Click += (sender, args) =>
             {
+                // Display a MessageBox to the user
+                // @todo Use a view framework to improve the UI/UX
```

Comment:
how would it improve UX ? All view frameworks come with baggage, i'd steer clear for such a simple thing.

Reply:
Discussion continues as issue https://github.com/teamtreedyn/Binoculars/issues/19 therefore marking this resolved.